packages/tc-ui/README.md

```diff
@@ -50,9 +50,16 @@ Base url for the treecreeper REST api
 
 Base url for the treecreeper GraphQL api
 
+##### clientId
+
+String to send to the API as the `Client-Id` header
```

Comment:
should this be `client-id`?

Reply:
headers aren't case sensitive. Most documentation I've seen that mentions headers uses Capital-Case... don't know why, but that seems to be the accepted convention (e.g. check out any Fastly documentation)